synapse/tests/test_cortex.py

```diff
@@ -2675,6 +2675,10 @@ class Bawk:
                 nodelist1 = [node.pack() for node in nodelist1]
                 self.eq(nodelist0, nodelist1)
 
+                # Try a nodeedits we might get from cmdr
+                cmdrnodeedits = s_common.jsonsafe_nodeedits(editlist[0])
+                await prox1.addFeedData('syn.nodeedits', [cmdrnodeedits])
```

Comment:
Is there an action from this edit we could validate as having been made to the cortex?